### DISCUSSION TITLE
argon spinner progress bars disappear on completion

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#21767] [Q&A]
I find this very weird and i'm constantly thrown off by it. I'd expect it to stay full and maybe flash a bit, but not shrink into oblivion

i'm not sure how well i can express how anticlimactic it feels, so i'll just leave a video below to show it

https://user-images.githubusercontent.com/36758269/209249724-d0a05770-daf3-4999-a5bb-371144c8997a.mp4

**Comment by JeromTWinL** [comment]
It maybe depend on your device's performace. (I'm not sure.)

**Reply by JeromTWinL** [reply]
And, can you tell me the beatmap you tested with?

**Reply by Walavouchey** [reply]
this... isn't a performance thing. it's designed that way (for some reason).

beatmap is [notch hell](https://osu.ppy.sh/beatmapsets/437683#osu/942356) if you're curious

**Reply by JeromTWinL** [reply]
I tested it right now, the spinner bonus is starting right after the progress bar finished. (And the skin flames are horrible.)

**Reply by JeromTWinL** [reply]
The tested video:


https://user-images.githubusercontent.com/84004577/209452926-ef12b6da-6263-4b3a-93ec-e5bec9803418.mp4

(Sorry for the bad audio quality, Recording app lowered it for premium.)

**Reply by Walavouchey** [reply]
i'm not talking about the spinner bonus appearing late or anything like that. just the "spinner progress bars disappear on completion" part.

**Reply by JeromTWinL** [reply]
But, it's completed and delayed to show the spinner bonus. (It's meant to show from 50, but it started from 400. I mean, i just noticed that only.), But you're right.

**Reply by JeromTWinL** [reply]
Sorry, At first, i just noticed spinner bonus is delaying, I didn't notice you meant "spinner progress bar disappeared", I now only understand what you meant (i didn't understand, as english isn't my first language.)

**Reply by JeromTWinL** [reply]
Maybe I'd not confused if you open this discussion as Suggestion.

**Comment by peppy** [comment]
I wanted to give some sense of when the spinner was completed so this was an experiment, but I agree that it doesn't feel great.

**Reply by JeromTWinL** [reply]
So, the spinner progress bar is just an experiment?

**Reply by peppy** [reply]
Not the progress bar itself, the fact it disappears and shrinks, as per what is mentioned in this discussion.

**Reply by JeromTWinL** [reply]
> I find this very weird and i'm constantly thrown off by it. **I'd expect it to stay full and maybe flash a bit, but not shrink into oblivion**
> 
> i'm not sure how well i can express how anticlimactic it feels, so i'll just leave a video below to show it
> 
>  20221223T005324-2333407036.mp4

You're going to do something for the spinner to stay on the progress bar?

**Comment by OninDynamics** [comment]
Instead of it disappearing, perhaps the progress bars should expand to form halves of a complete circle, flash, then fade away, leaving the thicker spinner-background-circle-thing under it.
